docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -174,6 +177,92 @@ For more details on getting started with the Synthetics Recorder, see <<syntheti
 
 image::images/synthetics-create-test-script-recorder.png[Elastic Synthetics Recorder after recording a journey and clicking Export]
 
+[discrete]
+[[synthetics-request-param]]
+== Make API requests
+
+You can use the `request` parameter to make API requests independently of the browser interactions.
+
+// Link to Playwright docs?
+// https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests
+
+NOTE: The `request` parameter is not intended to be used for writing pure API tests. Instead, it is a way to support
+writing plain HTTP requests in service of a browser-based test.
+
+Below is an example of how to use the `request` object to communicate with HTTP endpoints.
+The example demonstrates how to retrieve a token from an HTTP endpoint and use it in a subsequent webpage request.
```

Comment:
Do these examples look ok?

Reply:
Thanks @andrewvc. These examples were provided by @awahab07 in https://github.com/elastic/observability-docs/issues/2049#issuecomment-1201343573.  Should I remove them from here for the launch and open an issue to track incorporating what they illustrate into https://github.com/elastic/synthetics-demo instead?

Reply:
Yes, I'm OK with that

Reply:
Tracking in https://github.com/elastic/observability-docs/issues/2115.